.travis.yml

```diff
@@ -72,164 +82,229 @@ jobs:
       ANSIBLE_VERSION: "26"
+    name: functional tests shard 3/3, Ansible 2.6, Python 3.7
 
   - <<: *py-37
     env:
       <<: *env-functional-shard_1
       ANSIBLE_VERSION: "25"
+    name: functional tests shard 1/3, Ansible 2.5, Python 3.7
 
   - <<: *py-37
     env:
       <<: *env-functional-shard_2
       ANSIBLE_VERSION: "25"
+    name: functional tests shard 2/3, Ansible 2.5, Python 3.7
 
   - <<: *py-37
     env:
       <<: *env-functional-shard_3
       ANSIBLE_VERSION: "25"
+    name: functional tests shard 3/3, Ansible 2.5, Python 3.7
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_1
       ANSIBLE_VERSION: "27"
+    name: functional tests shard 1/3, Ansible 2.7, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_2
       ANSIBLE_VERSION: "27"
+    name: functional tests shard 2/3, Ansible 2.7, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_3
       ANSIBLE_VERSION: "27"
+    name: functional tests shard 3/3, Ansible 2.7, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_1
       ANSIBLE_VERSION: "26"
+    name: functional tests shard 1/3, Ansible 2.6, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_2
       ANSIBLE_VERSION: "26"
+    name: functional tests shard 2/3, Ansible 2.6, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_3
       ANSIBLE_VERSION: "26"
+    name: functional tests shard 3/3, Ansible 2.6, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_1
       ANSIBLE_VERSION: "25"
+    name: functional tests shard 1/3, Ansible 2.5, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_2
       ANSIBLE_VERSION: "25"
+    name: functional tests shard 2/3, Ansible 2.5, Python 3.6
 
   - <<: *py-36
     <<: *if-cron-or-manual-run-or-tagged
     env:
       <<: *env-functional-shard_3
       ANSIBLE_VERSION: "25"
+    name: functional tests shard 3/3, Ansible 2.5, Python 3.6
+
+  - <<: *py-27
+    env:
+      <<: *py27-env-functional-shard_1
+      ANSIBLE_VERSION: "27"
+    name: functional tests shard 1/3, Ansible 2.7, Python 2.7
 
   - <<: *py-27
     env:
+      <<: *py27-env-functional-shard_2
       ANSIBLE_VERSION: "27"
-      TESTS_TYPE: functional
+    name: functional tests shard 2/3, Ansible 2.7, Python 2.7
 
   - <<: *py-27
     env:
+      <<: *py27-env-functional-shard_3
+      ANSIBLE_VERSION: "27"
+    name: functional tests shard 3/3, Ansible 2.7, Python 2.7
+
+  - <<: *py-27
+    env:
+      <<: *py27-env-functional-shard_1
       ANSIBLE_VERSION: "26"
-      TESTS_TYPE: functional
```

Comment:
Removal of such env var WILL BREAK construction of `TOXENV` var. Don't do this.

Reply:
Isn'tit done in py27-env-functional-shard_1 ?

Reply:
Oh, right.. I overlooked it